src/plugin.js

```diff
@@ -154,18 +154,17 @@ export default class TableBlock {
    * A helper to get config value.
    * 
    * @param {string} configName - the key to get from the config. 
-   * @param {any} defaultValue - default value if config doesn't have passed key
    * @param {object} savedData - previously saved data. If passed, the key will be got from there, otherwise from the config
    * @returns {any} - config value.
    */
-  getConfig(configName, defaultValue = undefined, savedData = undefined) {
+  getConfig(configName, savedData = undefined) {
     const data = this.data || savedData;
```

Comment:
To achieve fixing config, I suggest not using `this.data` because `getConfig` is being used for making `this.data`.
```suggestion
    const data = savedData;
```